frontend/src/views/styles.module.css

```diff
@@ -0,0 +1,28 @@
+.problemPanel {
+    position: absolute;
+    top: 80px;
+    left: 0;
+    bottom: 0;
+    width: 50%;
+    display: inline-block;
+    background-color: #e3e3e3;
```

Comment:
Note for future reference: these colors might move into the Theme file once we come back and properly design everything